### DISCUSSION TITLE
Release and contributions

### DISCUSSION THREAD

**Discussion by sonnyp** [andyholmes/valent#305] [General]
@andyholmes thanks for GSConnect **and** Valent!

I have been using Valent for a while, but only now noticed that you opened up discussions.

I was wondering if you had plans for a release soon-ish and if you were now open to contributions?

To be frank, I have lost interest in maintaining/contributing to GSConnect knowing that its replacement is so far along already.

Cheers!

**Comment by andyholmes** [comment]
I don't think anything like a stable release will happen for some time. Unlike GSConnect, Valent has a plugin ABI/API which is not yet stable. Once I think I have all the base API I need hammered out, there'll probably be an alpha release to get some feedback before moving on. By 1.0, the codebase should be in a state where any potential problems can be fixed and features added without breaking ABI/API.

Contributions are open, but probably best to open a discussion about anything reasonably large. I have a pretty good idea of how I want to do most things, so I don't want anyone to waste work going in a different direction when we can discuss implementation first.

In contrast to GSConnect, I see Valent as more of a personal project I'm sharing, rather than a project I'm personally maintaining. That's just to say I'm happy to develop it as a community project and take contributions, but I'm going to be opinionated and not feel bad about that :slightly_smiling_face:. Fortunately Valent has a decent plugin API, so people can easily extend it, if they want.

**Reply by andyholmes** [reply]
Thanks for the sponsorship!
